sagemaker_neo_compilation_jobs/deploy_huggingface_model_on_Inf1_instance/inf1_bert_compile_and_deploy.ipynb

```diff
@@ -0,0 +1,1006 @@
+{
```

Comment:
<p>Grammar nit: <code>decrease -, while</code> - This seems like extra punctuation? Remove the comma?</p><p>Revise "cheapest" to "lowest-cost"? (2 occurrences)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>